docs/using/dbaas.md

```diff
@@ -23,6 +23,10 @@ From the left menu, select <i class="uil uil-database"></i> *DBaaS*.
 
 ### Add a Kubernetes cluster
 
+!!! caution alert alert-warning "Caution"
+    Ensure that you set *PMM Public Address* under <i class="uil uil-cog"></i> *Configuration* → <i class="uil uil-setting"></i> *Settings* → *Advanced Settings* before creating a Kubernetes cluster. Otherwise, PMM would not monitor the database clusters created in this Kubernetes cluster.
```

Comment:
database clusters wouldn't be monitored if public address wasn't before creating database cluster.
But we also have monitoring of kubernetes clusters and it requires to set public address before creating.